Documentation/design-docs/roll-forward-on-no-candidate-fx.md

```diff
@@ -0,0 +1,86 @@
+# Roll Forward On No Candidate Fx
+
+## Before changes
```

Comment:
Instead of _Before changes_, I would suggest you rename it to be "default behaviour".